### ISSUE TITLE
TST:  cannot import name _compiler from astropy.utils 

### ISSUE THREAD

**Issue by pllim** [astropy/astropy#9113]
Travis CI is broken on `master` and all the PRs. Example log: https://travis-ci.org/astropy/astropy/jobs/569993909

```
Traceback (most recent call last):

  File "<string>", line 1, in <module>

  File "/tmp/astropy-test-7ze_58fu/lib/python3.7/site-packages/astropy/__init__.py", line 367, in <module>

    _initialize_astropy()

  File "/tmp/astropy-test-7ze_58fu/lib/python3.7/site-packages/astropy/__init__.py", line 262, in _initialize_astropy

    from .utils import _compiler

ImportError: cannot import name '_compiler' from 'astropy.utils' (/tmp/astropy-test-7ze_58fu/lib/python3.7/site-packages/astropy/utils/__init__.py)
```

**Comment by pllim** [comment]
Looks like this affects other packages downstream too.

**Comment by pllim** [comment]
Fixed upstream at astropy/ci-helpers#411. Restart your jobs and they shall pass again (try read this in Gandalf's voice).

**Comment by pllim** [comment]
Caused by ContinuumIO/anaconda-issues#11195
